prowler/providers/aws/services/opensearch/opensearch_domain_fine_grained_access_control_enabled/opensearch_domain_fine_grained_access_control_enabled.metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "Provider": "aws",
+    "CheckID": "opensearch_domain_fine_grained_access_control_enabled",
```

Comment:
```suggestion
    "CheckID": "opensearch_service_domains_access_control_enabled",
```

Reply:
@abant07 I have done the changes but I cannot push to your branch, can you give me access, please?

Reply:
@abant07 can you change the name of the check, please?
And also, it is less conflicting for us if you open the PR from a new branch of your fork and not for the `master` branch.

Thanks!